test/k8s-integration/main.go

```diff
@@ -352,10 +355,12 @@ func generateGCETestSkip(clusterVersion string) string {
 	if v.LessThan(apimachineryversion.MustParseSemantic("1.16.0")) {
 		skipString = skipString + "|volumeMode\\sshould\\snot\\smount\\s/\\smap\\sunused\\svolumes\\sin\\sa\\spod"
 	}
-
 	if v.LessThan(apimachineryversion.MustParseSemantic("1.17.0")) {
 		skipString = skipString + "|VolumeSnapshotDataSource"
 	}
+	if platform == "windows" {
+		skipString = skipString + "|\\[LinuxOnly\\]|\\[Feature:.+\\][Ephemeral]"
```

Comment:
Do we need to skip Feature and Ephemeral? Skipping Feature means snapshot tests won't run on windows. Ephemeral I think is not run as part of the external storage test suite anyway.

Reply:
remove Feature and Ephemeral. I think the reason windows tests in presubmit remove Feature is to reduce running time. But periodic tests will test it.